app/views/admin/delivery_partners/index.html.erb

```diff
@@ -0,0 +1,41 @@
+<% page_data(title: "Delivery partners", breadcrumbs: @breadcrumbs) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+    <%= form_with method: :get do |f| %>
+      <%= f.govuk_text_field(
+        "q",
+        value: params[:q],
+        label: {
+          text: "Search for delivery partner",
+          size: "s",
+        },
+        hint: {
+          text: "Search by name",
+        },
+      ) %>
+
+      <%= f.govuk_submit "Search" %>
+    <% end %>
+    <% if params[:q].present? %>
+      <p class="govuk-body">
+        <strong><%= @pagy.count %></strong>
+        results found for "<%= params[:q] %>"
+      </p>
+    <% end %>
+
+    <%= govuk_table do |table|
+      table.with_head do |head|
+        head.with_row { |row| row.with_cell(text: "Delivery partner") }
+      end
+
+      table.with_body do |body|
```

Comment:
Do we want to hide this table and return `No results found` if there are no results?